_conferences/devfest-south-africa-2021.md

```diff
@@ -0,0 +1,14 @@
+---
+name: "GDG DevFest"
+website: https://devfest.co.za/
```

Comment:
The link is pointing to the 2020 event. Can you update it?

Reply:
Thanks, I changed  it to point to our community site.   Our external site isn't ready yet.

Reply:
Great 👍 Feel free to udpate it once you have the new URL